charts/0.5.0/pravega-operator/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+name: pravega-operator
```

Comment:
The Helm chart standard directory structure is:
```
charts/
  pravega-operator/
  <other charts>
```

What's the reasoning behind this new directory structure:
```
charts/
  0.4.0/
    pravega-operator/
    <other charts>
  0.5.0/
    pravega-operator/
    <other charts>
```

My thoughts on this is:
- It's going to break the Jenkins job which relies on the Helm standard directory structure (https://asd-nautilus-jenkins.isus.emc.com/job/pravega-operator/);
- Even if we managed to "fix" it, which would be going against the grain, we're now left with a Jenkins job that only works with that new structure and won't be able to retroactively build previous commits;
- The Jenkins job would need to be "re-fixed" every time a new numbered release comes out (i.e. a new parent directory);
- Old charts (in this case for `0.4.0`) should be kept in its own tag / branch. There's no reason to keep it along;

I strongly suggest reverting this decision before it's too late, else we're going to have some big problems. What I am suggesting is:
- Remove the 0.4.0 charts from there;
- Put the 0.5.0 charts back to its original location `charts/<chart name>/`, with no additional directory in-between;

Reply:
@pbelgundi thanks for reverting the chart directory structure change -- this looks a lot better! I will resolve this thread